src/app.js

```diff
@@ -25,6 +27,8 @@ const internals = {};
 const DEFAULT_GREETINGS_REGEX = /^(get started|good(morning|afternoon)|hello|hey|hi|hola|what's up)/i;
 const DEFAULT_HELP_REGEX = /^help\b/i;
 
+// TODO Extract this class into `index` and refactor `App` to be only the
+//      Express configuration and set up.
```

Comment:
Is that not the convention? I'm trying to line up underneath the TODO above to show that they're one thing.